sdk/identity/identity-web/README.md

```diff
@@ -0,0 +1,193 @@
+
+// We can also read the state if we configured one:
+const username = req.query["state"];
+
+// The credential can be re-created with the same parameters
+// or retrieved from memory, and then:
+const authenticationRecord = await credential.authenticate(scope, authorizationCode);
+
+// The authenticationRecord can be stored and re-used later:
+const credential2 = new WebRedirectCredential(
+  tenantId,
+  clientId,
+  redirectUri,
+  { authenticationRecord }
+);
+ 
+// After `authenticate` has been called, the credential can be used:
+const client = new KeyClient(vaultUrl, credential);
+```
+
+### Prerequisites
+
+- An [Azure subscription](https://azure.microsoft.com/free/).
+- An app registration configured with a redirect endpoint of the type `web`.
+
+To configure the Azure App Registration through the Azure Portal:
+
+- Go to the Active Directory.
+- Create a new App Registration, or open an existing one.
+- Go to the "Authentication" page of the App Registration.
+- Under "Supported Account Types", select `Accounts in any organizational directory (Any Azure AD directory - Multitenant)`.
+- Under "Platform Configurations":
+  - Users would need to add a platform and select "Web".
+  - Users then would add a specific URI to their server.
+  - If they're working with a localhost server, they may put something similar to `http://localhost:8080/path` as the redirect URI.
+  - Users would finally click "Configure".
+
+### Install the package
+
+```sh
+$ npm install --save-dev @azure/identity-web
+```
+
+#### Supported Environments
+
+Azure Identity plugins for JavaScript support stable (even numbered) versions of Node.js starting from v12. While the plugins may run in other Node versions, no support is guaranteed. `@azure/identity-web` **does not** support browser environments.
+
+## Key concepts
+
+If this is your first time using `@azure/identity` or the Microsoft identity platform (Azure Active Directory), we recommend that you read [Using `@azure/identity` with Microsoft Identity Platform](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/using-azure-identity.md) first. This document will give you a deeper understanding of the platform and how to configure your Azure account correctly.
+
+## Examples
+
+```javascript
+import { WebRedirectCredential } from "@azure/identity-web";
+
+app.get(
+  "/azureLogin",
+  async (req: express.Request, res: express.Response) => {
+    // Here we authenticate...
+    
+    // Then we make the credential.
+    const credential = new WebRedirectCredential(
+      tenantId,
+      clientId,
+      redirectUri
+    );
+
+    // We save the credential in an in-memory cache, or not...
+    // The sample will elaborate with recommended approaches.
+
+    // We set something that can identify the user as the state parameter.
+    const state = session.username; // or ID
+
+    // We get the authorize URL.
+    const authorizeUrl = credential.getRedirectUri(scope, {
```

Comment:
I think in the code you have this functio nas `getAuthorizeUrl` instead of `getRedirectUri`. We need to update all samples to use the correct method name